docs/usage/models.md

```diff
@@ -1,22 +1,22 @@
-The primary means of defining objects in *pydantic* is via models
-(models are simply classes which inherit from `BaseModel`).
+The primary means of defining objects in Pydantic is via models. Models are simply classes which inherit from
+`BaseModel`.
 
 You can think of models as similar to types in strictly typed languages, or as the requirements of a single endpoint
 in an API.
 
-Untrusted data can be passed to a model, and after parsing and validation *pydantic* guarantees that the fields
+Untrusted data can be passed to a model and, after parsing and validation, Pydantic guarantees that the fields
 of the resultant model instance will conform to the field types defined on the model.
 
 !!! note
-    *pydantic* is primarily a parsing library, **not a validation library**.
+    Pydantic is primarily a parsing library, **not a validation library**.
     Validation is a means to an end: building a model which conforms to the types and constraints provided.
 
-    In other words, *pydantic* guarantees the types and constraints of the output model, not the input data.
+    In other words, Pydanti guarantees the types and constraints of the output model, not the input data.
 
     This might sound like an esoteric distinction, but it is not. If you're unsure what this means or
     how it might affect your usage you should read the section about [Data Conversion](#data-conversion) below.
 
-    Although validation is not the main purpose of *pydantic*, you **can** use this library for custom [validation](validators.md).
+    Although validation is not the main purpose of Pydantic, you **can** use this library for custom [validation](validators.md).
```

Comment:
This may be a good place to address the fact that we use the word `validator` in many places to refer to something that might more accurately be described as "transforming". @samuelcolvin not sure how you are thinking about this these days..